BraintreeCore/src/main/java/com/braintreepayments/api/DeviceInspector.kt

```diff
@@ -16,41 +14,42 @@ import androidx.annotation.VisibleForTesting
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class DeviceInspector @VisibleForTesting internal constructor(
     private val appHelper: AppHelper,
-    private val uuidHelper: UUIDHelper,
     private val signatureVerifier: SignatureVerifier,
 ) {
+
     constructor() : this(
         AppHelper(),
-        UUIDHelper(),
         SignatureVerifier(),
     )
 
     internal fun getDeviceMetadata(
         context: Context?,
+        configuration: Configuration?,
         sessionId: String?,
-        integration: String?,
+        integration: String?
     ): DeviceMetadata {
         return DeviceMetadata(
-            platform = "Android",
-            platformVersion = Build.VERSION.SDK_INT.toString(),
-            sdkVersion = BuildConfig.VERSION_NAME,
-            merchantAppId = context?.packageName,
-            merchantAppName = getAppName(context),
+            appId = context?.packageName,
+            appName = getAppName(context),
+            clientSDKVersion = BuildConfig.VERSION_NAME,
+            clientOs = getAPIVersion(),
+            component = "braintreeclientsdk",
             deviceManufacturer = Build.MANUFACTURER,
             deviceModel = Build.MODEL,
-            devicePersistentUUID = uuidHelper.getPersistentUUID(context),
+            dropInSDKVersion = dropInVersion,
+            environment = configuration?.environment,
+            eventSource = "mobile-native",
+            integrationType = integration,
             isSimulator = isDeviceEmulator,
-            sessionId = sessionId,
-            integration = integration,
-            networkType = getNetworkType(context),
-            userOrientation = getUserOrientation(context),
-            appVersion = getAppVersion(context),
-            dropInVersion = dropInVersion,
-            isPayPalInstalled = isPayPalInstalled(context),
-            isVenmoInstalled = isVenmoInstalled(context)
+            merchantAppVersion = getAppVersion(context),
+            merchantId = configuration?.merchantId,
+            platform = "Android",
+            sessionId = sessionId
         )
     }
 
+    // Analytics payload no longer sends appInstalled info.
+    // Leaving logic for upcoming PaymentReady API implementation.
```

Comment:
We can just remove these checks if they aren't being used IMO

Reply:
Done [452daf4](https://github.com/braintree/braintree_android/pull/946/commits/452daf43b5529880bfca4c7b69e4586f882789ac)